x-pack/plugins/spaces/public/nav_control/components/spaces_menu.tsx

```diff
@@ -69,7 +69,6 @@ class SpacesMenuUI extends Component<Props, State> {
         id: 'xpack.spaces.navControl.spacesMenu.changeCurrentSpaceTitle',
         defaultMessage: 'Change current space',
       }),
-      watchedItemProps: ['data-search-term'],
```

Comment:
We already have an open issue to fix the Spaces Menu focus / keyboard navigation that broke in the EUI v55.0.1 upgrade 🙈 
We'll investigate any additional implications of this change in that issue, too: #130898

Reply:
There hopefully shouldn't be! This patch should (🤞) removed the need for `watchedItemProps` completely. Definitely feel free to ping if anything comes up!